tests/unit/core/api/test_categories.js

```diff
@@ -0,0 +1,35 @@
+/* global window */
```

Comment:
This whole test looks odd to me. It's doing things differently then we normally do them (now anyway), and maybe that's because the test was written so long ago. For example, I don't think we need `window` at all, but maybe I'm mistaken. Compare this to https://github.com/mozilla/addons-frontend/blob/2018.11.15-1/tests/unit/core/api/test_versions.js or https://github.com/mozilla/addons-frontend/blob/2018.11.15-1/tests/unit/amo/api/test_recommendations.js and you'll see what I mean.

I think, unless there's a compelling reason it needs to remain this way, it would be better to rewrite the test in the style of those referenced above.

Reply:
Yeah, I was sort of confused too when I moved the test case to the new file, but 🤷‍♂️ 

I don't think it is bad to mock `fetch` instead of `callApi` and I'd be tempted to only mock `fetch` but I guess the test suite for `callApi` is strong enough, so we can just continue to mock `callApi` everywhere. I'll update the test case here!